### ISSUE TITLE
About the performance in x2 model

### ISSUE THREAD

**Issue by NJU-Jet** [Zheng222/IMDN#20]
Thanks for your excellent work. 
But I hava a doublt about the results on x2 model. 
I can't get the results as you report in the article, so should I use L2 loss after 1000 epochs?
Thanks!

**Comment by Zheng222** [comment]
@NJU-Jet 
Hello, it doesn't need to use L2 loss. I recommend that training more iterations on larger learning rates, like 2e-4 and 1e-4. The performance improvement is mainly in these two learning rates.

**Comment by jinyige** [comment]
I use your X2 pre-train weight, test on Set5, the PSNR is  37.87,SSIM is 0.9593,they both lower than the papers? could you tell how to get the best result? Thank you very much.
